domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityComplexFieldACLValidatorImpl.java

```diff
@@ -73,13 +73,13 @@ private void validateCrudComplexParent(CaseFieldEntity caseField,
                                            CaseFieldEntityValidationContext caseFieldEntityValidationContext,
                                            ValidationResult validationResult,
                                            ComplexFieldACLEntity parentComplexFieldACLEntity) {
-        String parentUserRole = parentComplexFieldACLEntity.getUserRole() != null
-            ? parentComplexFieldACLEntity.getUserRole().getReference() : "";
+        String parentUserRole = parentComplexFieldACLEntity.getAccessProfile() != null
```

Comment:
rename variable.

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f

Reply:
Should probably still rename `parentUserRole`